### ISSUE TITLE
Display of locations

### ISSUE THREAD

**Issue by mparani70** [MedCyprus/MedCyprusEFES#22]
@gabrielbodard @emylonas 

Monument 1 <place xml:id="Acheleia" n="1"> appears last on EFES, regardless of numbering and the sorting order in locations.xml. All other monuments appear in the correct order. It needs to be moved to the top in the display.

**Comment by emylonas** [comment]
In the file ROOT/stylesheets/tei/indices.xsl l. 130, the monuments are being sorted incorrectly - 
```
<xsl:apply-templates select="doc[descendant::str[@name='index_item_type'][.='Monument']]"><xsl:sort select="lower-case(.)"/></xsl:apply-templates>
```
The sort is happening on the term "Monuments" which is the same in all entries, as this is restricted already to only locations that are monuments. It should be sorting either on the number (`index_number`) or the name (`index_item_name`. It looks like sorting a set of items on the same word produces odd results. As Solr seems to provide a results document that lists the items in the order in which they were entered in the locations file, I removed the sort, which has the desired effect (Acheleia is first). There is no guarantee that Solr will return items in a particular order, so if we start to see incorrect displays, it is possible to add the correct sort back in again. However, this seems to work for now.  

This means that the Repositories will also appear in the order in which they are entered in locations.xml. At the moment, two repositories don't appear because they don't have any inscriptions associated with them. 

Not closing the issue as I haven't pushed the changes yet.

**Comment by emylonas** [comment]
fixed with a29ad040dd2c7fbcad1df17ddc0ea38ec56a380e
